mandatory/1-fix-functions.js

```diff
@@ -9,9 +9,8 @@
   NOTE:Only make edits inside the function
 
 */
-
-function mood() {
-  let isHappy = true;
+function mood(ishappy) {
+  let isHappy = ishappy;
```

Comment:
no need to create a new variable here, you can just use the isHappy that is passed into the function in your if statement :)